### PR TITLE
Added Earnings Page

### PR DESCRIPTION
**Pull request by fru1tstand**
+ Chart.js to the rescue
+ Broke active tabs on split screen view, I'm aware and will fix (#45)
Part of #17

### CODE REVIEW

File: src/main/resources/templates/earnings.html [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+						label: 'Earnings',
+						data: [null, '6.23', '10.11', '14.82', '7.56'],
+						pointBackgroundColor: 'rgba(153, 102, 255, 0.2)',
+						pointHoverBackgroundColor: 'rgba(153, 102, 255, 0.5)',
+						fill: false,
+						yAxisID: 'y-axis-2'
+					}]
+				},
+				options: {
+					responsive: true,
```
Comment (jonathanpli) [inline src/main/resources/templates/earnings.html:58]: That's a convenient way to make a responsive chart. Small thing, but it only scales up when the screen size increases; subsequent resizing of the screen doesn't scale down. Will leave it up to you whether or not it's big enough of an issue to warrant fixing for the demo.

image

(increase screen size)

image

(decrease screen size)

image

**Review by jonathanpli — APPROVED** [review]

File: src/main/resources/templates/earnings.html [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/Chart.js/2.6.0/Chart.bundle.min.js"></script>
+<script>
+	(function() {
+		window.onload = function() {
+			let ctx = document.getElementById('earning-review-history').getContext('2d');
+			let myChart = new Chart(ctx, {
+				type: 'bar',
+				data: {
+					labels: ["May (this month)", "April", "March", "February", "January"],
```
Comment (jonathanpli) [inline src/main/resources/templates/earnings.html]: Can we have the order of the earnings on the chart go from January to May (chronological order)?

**Review by jonathanpli — CHANGES_REQUESTED** [review]

File: src/main/resources/templates/earnings.html (hunk 1 above)
Comment (fru1tstand) [reply src/main/resources/templates/earnings.html:58]: Found that issue as well. Don't think it's big enough issue for now. We won't be resizing the window once it's up for the demo.

File: src/main/resources/templates/earnings.html (hunk 2 above)
Comment (fru1tstand) [reply src/main/resources/templates/earnings.html]: It's in reverse chrono as I figured it'd be better to see the current month on the left with history on the right. Thoughts?

Comment (jonathanpli) [reply src/main/resources/templates/earnings.html]: I feel like logically for anything that's chronological, it makes more sense to see the most recent on the right side. From a UX standpoint, it's a little unintuitive (asked a couple housemates too what they thought).

We can leave it in reverse chronological order if you feel strongly about it, though!

Comment (fru1tstand) [reply src/main/resources/templates/earnings.html]: Nope. Was just curious on the reasoning. Changed.

**Review by jonathanpli — APPROVED** [review]

**Pull request merged**